pyaerocom/io/read_ebas.py

```diff
@@ -1215,7 +1286,7 @@ def _read_files(self, files, vars_to_retrieve, files_contain, constraints):
         This method is not supposed to be called directly but is used in
         :func:`read` and serves the purpose of parallel loading of data
         """
-        data_obj = UngriddedData()
+        data_obj = UngriddedData(num_points=1000000)
 
```

Comment:
Why is this initiated with this amount of points?

Reply:
Random choice, that was adapted from ReadGHOST. Was hoping this would speed up things a little, but did not test.